mne/preprocessing/maxwell.py

```diff
@@ -224,7 +224,7 @@ def _maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
                     head_pos=None, st_fixed=True, st_only=False,
                     mag_scale=100.,
                     skip_by_annotation=('edge', 'bad_acq_skip'),
-                    reconstruct='in', verbose=None):
+                    reconstruct='in', proc_msg=True, verbose=None):
```

Comment:
what's the difference between proc_msg and verbose?

Reply:
Sometimes you want some but not all messages. `proc_msg` makes it so that some only get displayed when `verbose` is sufficient *and* `proc_msg` is True.

Reply:
you'll need to add this to a docstring somewhere?

Reply:
no, it's in the private `_maxwell_filter` / just for internal use